packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -44,7 +46,8 @@ const propTypes = {
    */
   intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
   /**
-   * Allows user to set custom page label. _(usage note: User must pass translated text)_.
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)
+   * Allows user to set custom page label. _(usage note: User must pass translated text)_. It should not contain page number and total count details, which will lead to conflicts with build-in accessibility ARIA contexts. 
```

Comment:
@scottwilmarth I updated the description of the _pageLabel_ prop based on our discussion. Please let me know if it looks good to you